react/Viewer/Readme.md

```diff
@@ -48,29 +50,39 @@ initialState = {
   currentFileIndex: 0
 };
 
+const initialVariants = [
+  { infoPanel: false, navigation: true, toolbar: true, close: true }
+];
```

Comment:
it's certainly a nit, but is there a way to check the toolbar when we check "close" ? In order to show that the both options are linked together?

Reply:
J'ai rajouté un bloc de props pour la toolbar quand celle-ci est activée. J'ai mis le dépôt d'exemple à jour tu peux jeter un oeil.